pilot/notifications.rst

```diff
@@ -0,0 +1,40 @@
+.. _notifications:
+
+Notifications
+=======================================
+
+As a creator / owner of resources inside Ronin you may receive Email based notification depending on their state.
+These intend to alert you of possible under utilization of resources to help avoid unnecessary project spend.
+
+.. note::
+    All notifications will come from the address **noreply@ronin.cloud**, always be cautious when following links from external addresses!
+
+.. _instance_utilization:
+
+Instance Utilization
+---------------------------------------
+
+If you have received an email titled "Ronin Under Utilization Alert!" this means that Ronin has noticed an :term:`instance` in your project has gone for more than more than 24 hours with under 10% CPU utilization.
+This is designed to ensure you've not accidentally left a machine running idle. Unlike on-premise VM's that run 24/7 we recommend you shut down your instances when not in use, much like you would your own PC. 
+
+We do also understand this alert could be a false positive where your workloads are not CPU demanding but still require the machine be on for extended periods.
+If this is the case please get in touch and we can make the instance exempt from these alerts.
```

Comment:
It's more likely that the user has provisioned a far-too-big machine or doesn't understand how to use parallel processing, so it might be worth suggesting that they spin up a smaller machine if appropriate for that workload.

Reply:
I'm trying to avoid the discussion of 'right sizing' in this specific doc. I think it needs it's own page / paragraph to be referenced.
 
That said I'm not sure how to approach the topic as sizing is going to be very specific to the users needs and AWS' instance types are semi-regularly changing. It's hard not to just point to the AWS docs, which isn't helpful for a majority of people.

I fear the doc we make will end up being too simple. This is a topic we're eager to push over to Ronin to see if they can make the UX explain this better so that people are less likely to pick a silly instance for their workload.

Reply:
https://blog.ronin.cloud/selecting-machine/

Turns out they do have a doc for this. It's probably a good place to signpost people to.